orc8r/gateway/python/magma/common/misc_utils.py

```diff
@@ -234,6 +234,10 @@ def is_interface_up(interface):
     return netifaces.AF_INET in addr
 
 
+def get_default_interface():
+    gws = netifaces.gateways()
+    return gws['default'][netifaces.AF_INET][1]
```

Comment:
maybe you need to use get function in case `default` and `netifaces.AF_INET` and `1` doesn't exist